tutorials/EmailMagicLinks.md

```diff
@@ -0,0 +1,107 @@
+
+## Implementation
+The first thing you need to do is configure your application for deeplinks, as they are the fundamental aspect of the email magic link flow. For more information on configuring deeplinks, read our [Deeplinks Tutorial](./Deeplinks.md).
+
+Next, call the appropriate method with any optional parameters you desire:
+```swift
+import StytchCore
+
+// Create the parameters
+let parameters = StytchClient.MagicLinks.Email.Parameters.init(
+    email: "user@email.com",
+    // The following parameters are all optional
+    loginMagicLinkUrl: URL(string: "my-app://stytch-auth/login"), // The URL to be directed to on login. Will use your default, if not provided
+    loginExpiration: .defaultSessionDuration, // the duration after which the login url should expire
+    loginTemplateId: "my-login-template", // a custom template for login emails, if you've configured one
+    signupMagicLinkUrl: URL(string: "my-app://stytch-auth/signup"),  // The URL to be directed to on user creation. It is ignored on send requests. Will use your default, if not provided
+    signupExpiration: .defaultSessionDuration, // the duration after which the login url should expire
+    signupTemplateId: "my-signup-template" // a custom template for signup emails, if you've configured one. It is ignored on send requests
+)
+
+func loginOrCreate(parameters: StytchClient.MagicLinks.Email.Parameters) {
+    Task {
+        do {
+            // Requests an email magic link for a user to log in or create an account depending on the presence and/or status of an existing account.
+            let response = try await StytchClient.magicLinks.email.loginOrCreate(parameters: parameters)
+            print(response)
+        } catch {
+            print(error.errorInfo)
+        }
+    }
+}
+
+func send(parameters: StytchClient.MagicLinks.Email.Parameters) {
+    Task {
+        do {
+            // Requests an email magic link for an existing user to log in or attach the included email factor to their current account.
+            let response = try await StytchClient.magicLinks.email.send(parameters: parameters)
+            print(response)
+        } catch {
+            print(error.errorInfo)
+        }
+    }
+}
+```
+
+Lastly, once you receive the deeplink, authenticate the token (this is an expansion on the [parsing deeplinks tutorial.](./Deeplinks.md)):
+```swift
+import StytchCore
+
+func handle(url: URL) {
+    Task {
+        do {
+            switch try await StytchClient.handle(url: url, sessionDuration: 5) {
+            case let .handled(response):
+                print("handled: \(response.session) - \(response.user)")
+            case .notHandled:
+                print("not handled")
+            case let .manualHandlingRequired(tokenType, token):
+                if tokenType == .magicLinks {
```

Comment:
You actually don't need to do this for EML when using the `handle` method. The only thing that returns `manualHandlingRequired` is the password reset by email flow. The EML flow will be fully authenticated by the `handle` method and return a `.handled(response)` (where the response is the auth response)